tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -11,6 +11,14 @@ namespace Datadog.Trace.Ci.Coverage.Attributes
         public CoveredAssemblyAttribute() { }
     }
 }
+namespace Datadog.Trace.Ci.Coverage.Metadata
+{
+    public abstract class ModuleCoverageMetadata
+    {
+        protected readonly int[][] Metadata;
+        protected ModuleCoverageMetadata() { }
+    }
+}
```

Comment:
This probably doesn't need to be public right?

Reply:
Sadly nope, this abstract class is inherited by the new `Metadata` type we emit in the target assembly. So, to bypass all visibility checks this needs to be public.